src/OSPSuite.InstallationValidator.Core/Services/BatchComparisonTask.cs

```diff
@@ -58,13 +58,20 @@ public Task<BatchComparisonResult> StartComparison(string folderPath, Cancellati
          {
             _validationLogger.AppendLine(Logs.ComparingFilles(file));
             var fileComparison = await compareFile(file, folderPath1, folderPath2, token);
+            fileComparison.OutputComparisonResults.Each(x => captionOutputs(x, folderPathCaption1, folderPathCaption2));
```

Comment:
Why is this required so deep in the hierarchy? Should not it be just a prop of the BatchComparisonResult?